omego/artifacts.py

```diff
@@ -252,6 +256,28 @@ def __init__(self, args):
             base_url + a.find("relativePath").text for a in artifacts]
         self.find_artifacts(artifacturls)
 
+    def guess_ci_server(self, ci):
+        """
+        If this is a short name (no protocol, no dots) guess the CI server
+        (i.e. devspace)
+        """
+        if re.match(r'\w+://', ci) or '.' in ci:
```

Comment:
It's now `https` only. Since this is intended to be a handt shortcut it's not meant to be exhaustive, anyone needing something more complicated such as a scheme can type the full thing out.